tests/lib/snaps/test-snapd-sh/meta/snap.yaml

```diff
@@ -1,6 +1,15 @@
 name: test-snapd-sh
 summary: A no-strings-attached, no-fuss shell for writing tests
 version: 1.0
+
+plugs:
+    personal-files:
+        read: [$HOME/.testdir1, $HOME/.testfile1, $HOME/testfile1, $HOME/testdir1]
```

Comment:
Is it guaranteed that `$HOME` expanded has no trailing slash? If it may have a trailing slash then the paths won't be clean. I wonder if we should handle that (cleaning $HOME ourselves)